internal-v2.nix

```diff
@@ -422,23 +321,8 @@ rec {
   patchedLockfile = sourceOptions: parsedLockFile:
     let
       patched = patchLockfile sourceOptions parsedLockFile;
-      replacement = writeText "packages_hashes.sed" (lib.concatMapStrings
-        (dep: lib.optionalString (dep ? id) ''
-          s @${dep.id}@ ${lib.removeSuffix "\n" (builtins.readFile dep.hash)} g;
```

Comment:
Is this what you mean by IFD? we build dep in order to produce our package-lock.json?

But if dep can be cached is this still considered an IFD?